overlord/snapstate/snapmgr_test.go

```diff
@@ -641,7 +642,7 @@ func (s *snapmgrTestSuite) TestInstallRunThrough(c *C) {
 		},
 		{
 			op:   "copy-data",
-			name: "/snap/some-snap/11",
+			name: fmt.Sprintf("%s/some-snap/11", dirs.SnapMountDir),
```

Comment:
We don't need to reassure that this path is not hardcoded _on every single place_ we use snap.Info. If you want to make sure the path is right, let's please have _one test_ that checks the path is right in the specific place you are concerned about, rather than duplicating that logic in hundreds of places.